src/main/webapp/style.css

```diff
@@ -120,3 +120,64 @@ html * {
       margin-right: 0;
   }
 }
+
+#calendar{
+  max-height: 85vh;
+  max-width: 100vw;
+}
+
+.popover{
+  width: 30vw !important;
+  min-width: 448px !important;
+  height: auto !important;
+  max-width: 30vw !important;
+  max-height: 70vh !important;
```

Comment:
Can you escape !important? I think, it's bad style.

Reply:
I don't think so, I will try, but these overwrite some settings from fullcalendar.

Reply:
Update: it worked